export/distro/format.go

```diff
@@ -167,3 +168,11 @@ func (af azureFormatter) Format(event *models.Event) []byte {
 	}
 	return msg
 }
+
+type noopFormatter struct {
+}
+
+func (noopTr noopFormatter) Format(event *models.Event) []byte {
+	var ret []byte
+	return ret
```

Comment:
No need for declaring a var. You can `ret []byte` directly.

BTW. why is this a separate function? When you need empty byte array in the code, can't you just declare the one over there?

Reply:
@drasko They just want to create a type that satisfies the ```Formatter``` interface. It does nothing because they only use the ```Sender``` interface of the distro pipeline. 
How do you recommend to do that without defining a type?

Reply:
Looks like "return []byte" is not legal, I will change it to "return []byte{}"
As @feclare said, the separate function is to match the Formatter interface we already have for formatter. I return a byte array to make the function in the interface. It's empty because this formatter is meant to do nothing, that why it's called noop.